components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -3535,6 +3535,81 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  ######################################################
+  # The 'Role Alias' Collection API Resource
+  ######################################################
+  /system-scopes/role-aliases:
+    #-----------------------------------------------------
+    # Retrieve role alias mappings for system scope roles
+    #-----------------------------------------------------
+    get:
+      summary: Retreive role alias mappings
+      operationId: systemScopesRoleAliasesGet
+      security:
```

Comment:
x-wso2-curl, x-wso2-request, x-wso2-response should be defined

Reply:
@rmsamitha we need to use x-examples instead. Let's define it for now.

On a separate note, we need to rethink the way we define examples in REST APIs.

Reply:
As per https://github.com/wso2/product-apim/issues/8231, x-wso2-curl, x-wso2-request, and x-wso2-response will be added as x-examples.
[Example for x-examples.](https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml#L128)

Reply:
+1 for x-examples